### PR TITLE
20210423 xsrf token refresh

### PR DESCRIPTION
**Pull request by jimbojr55**
"GIT"

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,15 +27,10 @@ before_install:
 install:
```
Comment (jimbojr55) [inline .travis.yml:27]: "GET"

Comment (jimbojr55) [reply .travis.yml:27]: GET:

**Review by jimbojr55 — COMMENTED** [review]
GET/.GET\.

File: .travis.yml (hunk 1 above)
Comment (jimbojr55) [reply .travis.yml:27]: @jimbojr55

**Comment by jrobbins** [comment]
It looks like you are just experimenting with how GitHub pull requests work.

**Pull request closed without merging**